block-storage-attacher/watcher/pv_watcher.go

```diff
@@ -298,7 +299,8 @@ func UpdatePersistentVolume(volume config.Volume, pv *v1.PersistentVolume) (bool
 				line = space.ReplaceAllString(line, " ")
 				line_parts := strings.Split(string(line), " ")
 				lgr.Info("Line: ", zap.Strings("LINE", line_parts))
-				if len(line_parts) >= 2 {
+				// We ignore the orphan multipaths
+				if (len(line_parts) >= 3) && (strings.TrimSpace(line_parts[2]) != "orphan") {
```

Comment:
does the devicepath shows as  "orphan" ?
can't we have just check for only specific 2 paths with 2 target IP and ignore the rest ?  Target IPs might be available part of the PV Spec